### PR TITLE
Initial text on mapping to ACE.

### PR DESCRIPTION
**Pull request by malishav**

### CODE REVIEW

File: draft-selander-ace-ake-authz.md [hunk 1]
```diff
@@ -369,6 +369,113 @@ TODO: The voucher response may contain a "Voucher-info" field as an alternative
...
+   |           LAKE Message 3        |                     |
+   |-------------------------------->|                     |
+
+~~~~~~~~~~~
+{: #fig-mapping-ace title="Overview of the protocol mapping to ACE" artwork-align="center"}
+
+RS proactively sends the AS Request Creation Hints message to C to signal the information on where C can reach the AS.
+RS piggybacks the AS Request Creation Hints message using Auxiliary Data of the LAKE message 1.
+Before continuing the LAKE handshake, based on the AS Request Creation Hints information, C sends a POST request to the token endpoint at the AS requesting the access token.
+The AS issues to C a Bearer token that is cryptographically protected based on the secret shared between the AS and RS.
```
Comment (mcr) [inline draft-selander-ace-ake-authz.md]: I would rather we wrote: "AS issues an object to C that is ...  It is encoded using the Bearer Token container."

File: draft-selander-ace-ake-authz.md [hunk 2]
```diff
@@ -369,6 +369,113 @@ TODO: The voucher response may contain a "Voucher-info" field as an alternative
...
+AD1:
+{
+    "AS" : "coaps://as.example.com/token",
+    "audience": << h'73',h'737570657273...' >>
+}
+~~~~~~~~~~~
+
+## Client-to-AS Request
+
+Protocol that provides the secure channel between C and the AS is out-of-scope.
```
Comment (mcr) [inline draft-selander-ace-ake-authz.md]: The Protocol that..

**Comment by gselander** [comment]
I made changes where Michael commented.

**Comment by malishav** [comment]
Thanks, I was just about to address those!

**Pull request merged**